packages/mui-material/src/Backdrop/Backdrop.d.ts

```diff
@@ -96,7 +110,7 @@ export interface BackdropTypeMap<
   AdditionalProps = {},
   RootComponent extends React.ElementType = 'div',
 > {
-  props: AdditionalProps & BackdropOwnProps;
+  props: AdditionalProps & BackdropOwnProps & BackdropSlotsAndSlotProps;
```

Comment:
@sai6855 this is were I made the mistake with the `Alert`, right? (https://github.com/mui/material-ui/pull/41324)

The fix here would be to remove `& BackdropSlotsAndSlotProps` from this line and instead do

```ts
export interface BackdropOwnProps extends Partial<Omit<FadeProps, 'children'>>, BackdropSlotsAndSlotProps {
// ...
```

on line 45.

Reply:
Yup correct 👍

Reply:
Thanks!